packages/google-compute-engine-oslogin/utils/oslogin_utils.cc

```diff
@@ -307,6 +306,72 @@ bool ValidatePasswd(struct passwd* result, BufferManager* buf,
   return true;
 }
 
+bool ParseJsonToUsers(const string& json, std::vector<string> *result) {
```

Comment:
ParseJsonToGroupMembers ?

Reply:
The JSON field name is `usernames` and we pass this around as variable `users`.

Reply:
The function name is too generic and it would not work any json. Should we rename it to ParseGroupMembersResponseToUsers or something which is not this generic? Wdyt?

Reply:
The resource we are getting is users, from `/users?groupname=` or `/users?pagesize=`, so it should be ParseUsersResponseToUsers in that case. I don't mind, but if we decide to change this I think we'll have to change all the ParseJsonTo* functions for consistency.